src/core/hle/service/ir/extra_hid.cpp

```diff
@@ -228,24 +226,21 @@ void ExtraHID::OnReceive(const std::vector<u8>& data) {
 }
 
 void ExtraHID::SendHIDStatus() {
-    if (is_device_reload_pending.exchange(false))
-        LoadInputDevices();
-
     constexpr int C_STICK_CENTER = 0x800;
     // TODO(wwylele): this value is not accurately measured. We currently assume that the axis can
     // take values in the whole range of a 12-bit integer.
     constexpr int C_STICK_RADIUS = 0x7FF;
 
     float x, y;
-    std::tie(x, y) = c_stick->GetStatus();
+    // std::tie(x, y) = c_stick->GetStatus();
```

Comment:
You can always request the shared pointer from the emulated controller from anywhere to read input data